sanic/app.py

```diff
@@ -3,7 +3,14 @@
 import os
 import re
 
-from asyncio import CancelledError, Protocol, ensure_future, get_event_loop
+
+try:
```

Comment:
maybe we can have this in compat.py module, and don't need try/except blob in multiple places